src/Compatibility/Core/src/iOS/Renderers/FrameRenderer.cs

```diff
@@ -133,7 +133,10 @@ public virtual void SetupLayer()
 		public override void LayoutSubviews()
 		{
 			if (_previousSize != Bounds.Size)
+			{
 				SetNeedsDisplay();
+				this.UpdateBackgroundLayer();
```

Comment:
If you're modifying the compatibility renderer should this call get moved up to here?

https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Compatibility/Handlers/iOS/VisualElementRenderer.cs#L10